.yamato/project.metafile

```diff
@@ -33,7 +33,6 @@ projects:
       - name: com.unity.multiplayer.samples.coop
         path: Packages/com.unity.multiplayer.samples.coop
     test_editors:
-      - 2021.1
-      - 2021.2
       - 2020.3
+      - 2021.2
       - trunk
```

Comment:
There's a chance we'll remove trunk eventually if things fail, but I'm curious to see it work

Reply:
(you can resolve once seen)

Reply:
Resolving these as now we're only targeting 2021.3.